### PR TITLE
feat: expand cell

### PR DESCRIPTION
**Pull request by vcellu**
Recommended testing
1. clone sn-table into a new folder
2. in sn-table checkout work/ongoine
3. `yarn`
4. `yarn setup:rn:dev`
5. `yarn build:rn:watch`
6. launch vscode iOS Hermes

### CODE REVIEW

File: src/components/Cell.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  textCol2: {
+    fontWeight: 'bold',
+  },
+  miniChart: {
+    minHeight: 96,
```
Comment (gruskal) [inline src/components/Cell.tsx:87]: Why 96? The frame seems bigger than the chart
Screen Shot 2022-10-24 at 16 54 48

**Review by gruskal — APPROVED** [review]
Looks good. Awesome to see that we're able to use React for the expand table!

File: src/components/Cell.tsx (hunk 1 above)
Comment (vcellu) [reply src/components/Cell.tsx:87]: It's a minHeight, 96 is a factor of 8 and looked the nicest.

**Pull request merged**